core/src/main/java/hudson/model/UsageStatistics.java

```diff
@@ -212,7 +215,7 @@ public Permission getRequiredGlobalConfigPagePermission() {
     public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
         try {
             // for backward compatibility reasons, this configuration is stored in Jenkins
-            Jenkins.get().setNoUsageStatistics(json.has("usageStatisticsCollected") ? null : true);
+            Jenkins.get().setNoUsageStatistics(json.has("usageStatisticsCollected") ? null : Boolean.TRUE);
```

Comment:
tidy up.  (given this code was updated in previous commits it had been changed) this want's a `Boolean` not a `boolean` so avoids autoboxing and an IDE warning.